src/ui/components/navigation/navigationcomponent.js

```diff
@@ -236,12 +238,31 @@ export default class NavigationComponent extends Component {
       this.refitNav();
       DOM.on(DOM.query(this._container, '.yxt-Nav-more'), 'click', this.toggleMoreDropdown.bind(this));
     }
+    DOM.queryAll(this._container, '.js-yxt-navItem').forEach((selector, index) => {
+      DOM.onClickOrEnter(selector, this.reportFollowUpQueryEvent.bind(this, index));
+    });
   }
 
   onUnMount () {
     this.unbindOverflowHandlers();
   }
 
+  reportFollowUpQueryEvent (tabIndex) {
+    const searcher = tabIndex === 0 ? Searcher.UNIVERSAL : Searcher.VERTICAL;
+    const tabs = this.getState('tabs');
+    const tabUrlString = tabs?.[tabIndex]?.url || '';
+    let sameOriginForTabUrl = true;
+    if (tabUrlString.startsWith('http')) {
+      const tabUrl = new URL(tabUrlString);
```

Comment:
do we need a polyfill for URL?

Reply:
ah ie11, I will just parse the origin directly then